package.json

```diff
@@ -23,6 +23,9 @@
     "rollup": "rollup -c",
     "test": "jest --coverage && eslint src",
     "test:watch": "jest --coverage --watchAll --json --outputFile=testOutput.json",
+    "test-storybook": "test-storybook",
+    "test-storybook:watch": "test-storybook --watch",
+    "test-storybook:browsers": "test-storybook --browsers firefox chromium webkit",
```

Comment:
New scripts to run test-runner locally, in watch mode and simulating interactions with different browsers.